nomenclature/definitions/variable/technology/electricity-grid.yaml

```diff
@@ -1,25 +1,90 @@
 # List of variables related to the electricity grid
+# description of variables related to electricity demand
+Value of Lost Load|Electricity:
+   description: Cost of demand-not-served. Cost of load curtailment. Value of Lost Load (VoLL)
+   unit: €/MWh
 
-Network|Electricity|Demand|Reserve|Frequency Containment:
-   description: demand in frequency containment reserve (FCR)            
+Demand not served|Electricity:
+   description: Demand-not-served.
+   unit: MWh
+
+Reserve|Electricity|Requirement|Frequency Containment:
+   description: Demand in frequency containment reserve (FCR)
    unit: MW
-   
-Network|Electricity|Demand|Reserve|Automatic Frequency Restoration:
-   description: demand in automatic frequency restoration reserve (AFRR)             
+
+Reserve|Electricity|Requirement|Automatic Frequency Restoration:
+   description: Demand in automatic frequency restoration reserve (AFRR)
    unit: MW
-   
-Network|Electricity|Demand|Inertia:
-   description: demand in inertia            
+
+Network|Electricity|Requirement|Inertia:
+   description: Demand in inertia
    unit: second
-   
-Maximum Flow|Electricity|Grid:
-   description: maximum flow of an interconnection between 2 regions
-   unit: MW
-   
-Minimum Flow|Electricity|Grid:
-   description: minimum flow of an interconnection between 2 regions
+
+# description of variables related to transmission lines
+Network|Electricity|Active Power Flow:
+   description: Active power flow through a line
    unit: MW
-   
-Impedance|Electricity|Grid:
-   description: impedance of an interconnection between 2 regions
+
+Network|Electricity|Investment:
+   description: Investment decision. Note that it is a output data from model and if it makes a product with the investmen cost of the line, we get the investment in monetary unit.
+   unit: [Yes, No]
+
+Network|Electricity|Impedance:
+   description: Impedance of an interconnection between 2 regions
    unit: S
+
+Network|Electricity|Investment:
+   description: Overnight investment (capital) cost in transmission lines of the power network
+   unit: billion US$2010/yr
+
+Network|Electricity|Fixed Charge Rate:
+   description: Fixed charge rate to annualize the overnight investment cost. The Fixed Charge Rate (FCR) is the percentage of the total investment cost that is required over the project life per year to cover the minimal annual revenue requirements.
+   unit:
+
+Network|Electricity|Line Type:
+   description: Line type related to alternating current (AC) and direct current (DC)
+   unit: [AC, DC]
+
+Network|Electricity|Line Losses:
+   description: Half ohmic losses of a line because losses are divided and located at each of the connection points of the transmission line.
```

Comment:
A line is a connection from point a to point b. In my opinion, this variable should reflect the entire losses on that line. How it is implemented (half of losses accounted for at each bus, or 100% at one bus) is a model design choice.

Reply:
Got it. Thx.